### PR TITLE
Remove unused plugins convert metadata

### PR DESCRIPTION
**Pull request by b00y0h**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -33,39 +33,16 @@ if (!spaceId || !accessToken) {
...
-        headers: {}, // option to add more headers. `Link` headers are transformed by the below criteria
-        allPageHeaders: [], // option to add headers for all pages. `Link` headers are transformed by the below criteria
-        mergeSecurityHeaders: true, // boolean to turn off the default security headers
-        mergeLinkHeaders: true, // boolean to turn off the default gatsby js headers
-        mergeCachingHeaders: true, // boolean to turn off the default caching headers
-        transformHeaders: (headers) => headers, // optional transform for manipulating headers under each path (e.g.sorting), etc.
-        generateMatchPathRewrites: true, // boolean to turn off automatic creation of redirect rules for client only paths
-      },
-    },
+    'gatsby-plugin-netlify',
```
Comment (b00y0h) [inline gatsby-config.js:45]: removed options that were set to default anyway

File: gatsby-config.js [hunk 2]
```diff
@@ -92,25 +69,6 @@ module.exports = {
         defaultQuality: 50,
       },
     },
-    {
-      resolve: `gatsby-plugin-manifest`,
```
Comment (b00y0h) [inline gatsby-config.js:96]: really the only benefit we were getting from this was the favicon. we can find another way to do this

File: package.json [hunk 3]
```diff
@@ -73,7 +64,7 @@
   "license": "MIT",
   "scripts": {
     "build": "gatsby build",
-    "netlifyBuild": "netlify build",
+    "netlifyBuild": "npm run clean && netlify build && netlify deploy",
```
Comment (b00y0h) [inline package.json:67]: shoot, meant to set this back. i'll repush

File: src/components/common/SEO/getImageUrls.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const getImageUrls = ({ images, siteUrl }) =>
```
Comment (b00y0h) [inline src/components/common/SEO/getImageUrls.js:1]: for future work

File: src/data/queries/fragments.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { graphql } from 'gatsby'
```
Comment (b00y0h) [inline src/data/queries/fragments.js:1]: for future work

**Review by andreieab — COMMENTED** [review]
No comments

**Review by andreieab — APPROVED** [review]

**Review by andreieab — APPROVED** [review]

**Pull request merged**